code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -11,11 +11,11 @@
 	usable_hands = 0 //Populated on init through list/bodyparts
 	mobility_flags = MOBILITY_FLAGS_CARBON_DEFAULT
 	blocks_emissive = NONE
-	///List of [/obj/item/organ] in the mob. They don't go in the contents for some reason I don't want to know.
+	///List of [/obj/item/organ/internal] in the mob. They don't go in the contents for some reason I don't want to know.
 	var/list/internal_organs = list()
```

Comment:
why not explicitly type it for spacemandmm eg var/list/obj/item/organ/internal_organs

Reply:
done